templates/resources/arm/aks.json

```diff
@@ -0,0 +1,80 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"clusterName": {
+			"type": "string"
+		},
+		"dnsPrefix": {
+			"type": "string"
+		},
+		"clusterLocation": {
+			"type": "string"
+		},
+		"servicePrincipalId": {
+			"type": "securestring"
+		},
+		"servicePrincipalKey": {
+			"type": "securestring"
+		},
+		"agentCount": {
+			"defaultValue": 3,
+			"type": "int"
+		},
+		"agentVMSize": {
+			"defaultValue": "Standard_D2_v2",
+			"type": "string"
+		},
+		"kubernetesVersion": {
+			"defaultValue": "1.11.5",
+			"type": "string"

```

Comment:
Shashank mentioned that we should try out by removing this and I like that idea.

Reply:
I removed it.